src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -992,7 +992,7 @@ MemberName resolveOrFail(byte refKind, MemberName m,
             if (result.isResolved())
                 return result;
             ReflectiveOperationException ex = result.makeAccessException();
-            if (ex instanceof IllegalAccessException)  throw (IllegalAccessException) ex;
+            if (ex instanceof IllegalAccessException iae)  throw iae;
```

Comment:
```suggestion
            if (ex instanceof IllegalAccessException iae) throw iae;
```

Reply:
Don't know why, but sometimes JDK devs leaves 2 spaces for certain breaks (like Joe Darcy tends to leave 2 for sentence breaks in Javadocs). Can anyone explain why? In MemberName in particular, I see a mix of 2 and 1 spaces for single-line if statements, so I will take your suggestion.